index.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Compiles basic twig templates for patternlab iframe wrapper
+ *
+ * USAGE:
+ * var patternlabWrapper = require('patternlab-wrapper');
+ *
+ * patternlabWrapper({
+ *   dest: './app/public',
+ *   patternPaths: {"atoms":{"colors":"00-atoms-01-global-00-colors","redirect":"00-atoms-01-global-01-redirect",...}}
+ *   patternNavHTML: '<ul class="patterns-menu sg-nav">
+    <li class="sg-nav-"><a class="sg-acc-handle">base</a>
+        <ul class="sg-acc-panel">
+
+            <li class="sg-nav-"><a class="sg-pop" href="app/bower_components/pattern-library/patterns/base/blockquote">Blockquote</a></li>
+            <li class="sg-nav-"><a class="sg-pop" href="app/bower_components/pattern-library/patterns/base/body">Body</a></li>
+        </ul>
+    </li>
+    </ul>'
+ * });
+ *
+ */
+
+/**
+ *
+ * @type {Twig.exports|exports}
+ */
+
+var Twig = require('twig');
+var fs = require('fs');
+var fsextra = require('fs-extra');
+
+var patternlabFiles = [
+  'styleguide', 'js', 'images', 'fonts', 'data'
+];
+
+/**
+ *
+ * @type {{dest: string, patternNav: string, patternNavTemplate: string, ishControls: string, ishControlsTemplate: string, patternPaths: {base: Array}, src: string}}
+ */
+var optionsDefault = {
+  dest: './public',
+  patternNavHTML: '',
+  patternNavTemplate: './_patternlab-files/partials/patternNav.twig',
+  ishControlsHTML: '',
+  ishControlsTemplate: './_patternlab-files/partials/ishControls.twig',
+  patternPaths: {base: []},
+  src: './_patternlab-files'
+};
+
+function patternlabWrapper(options) {
+  options = options || {};
+  options.patternNavHTML = options.patternNavHTML || Twig.twig({data: fs.readFileSync(optionsDefault.patternNavTemplate, 'utf8')}).render();
+  options.ishControlsHTML = options.ishControlsHTML || Twig.twig({data: fs.readFileSync(optionsDefault.ishControlsTemplate, 'utf8')}).render();
+  options.patternPaths = options.patternPaths || optionsDefault.patternPaths;
+  options.dest = options.dest || optionsDefault.dest;
+  options.src = options.src || optionsDefault.src;
+
+  var indexTemplate = Twig.twig({data: fs.readFileSync('./_patternlab-files/index.twig', 'utf8')});
```

Comment:
I would suggest to move all shared instances like twig, custom helper functions and shared npm modules to something like ```pattern-library/core``` or ```pattern-library/utils``` in this case. I think pattern-importer dependency just for twig instance is unnecessary here.